guide/latest-release/supported-platforms.markdown

```diff
@@ -13,31 +13,31 @@ for all supported platforms and [binary packages for popular Linux distributions
 
 ## Hub
 
-| Platform    | Versions                   | Architecture |
-|:-----------:|:--------------------------:|:------------:|
-| CentOS/RHEL | 7, 8.1+                    | x86-64       |
-| Debian      | 9, 10, 11, 12              | x86-64       |
-| Debian      | 11, 12                         | arm64        |
-| Ubuntu      | 16.04, 18.04, 20.04, 22.04 | x86-64       |
-| Ubuntu      | 22.04                      | arm64        |
+| Platform    | Versions                          | Architecture |
+|:-----------:|:---------------------------------:|:------------:|
+| CentOS/RHEL | 7, 8.1+                           | x86-64       |
+| Debian      | 9, 10, 11, 12                     | x86-64       |
+| Debian      | 11, 12                            | arm64        |
+| Ubuntu      | 16.04, 18.04, 20.04, 22.04, 24.04 | x86-64       |
+| Ubuntu      | 22.04, 24.04                      | arm64        |
 
 Any supported host can be a policy server in Community installations of CFEngine.
 
 ## Clients
 
-| Platform    | Versions                   | Architectures   |
-|:-----------:|:--------------------------:|:---------------:|
-| AIX         | 7.1, 7.2                   | PowerPC         |
-| CentOS/RHEL | 6, 7, 8.1, 9               | x86-64          |
-| Debian      | 9, 10, 11, 12              | x86-64          |
-| Debian      | 11, 12                         | arm64           |
-| HP-UX       | 11.31+                     | Itanium         |
-| SLES        | 11, 12, 15                 | x86-64          |
-| Solaris     | 11                         | UltraSparc      |
-| Solaris     | 10                         | UltraSparc, x86 |
-| Ubuntu      | 16.04, 18.04, 20.04, 22.04 | x86-64          |
-| Ubuntu      | 22.04                      | arm64           |
-| Windows     | 2012, 2016, 2019           | x86-64, x86     |
+| Platform    | Versions                          | Architectures   |
+|:-----------:|:---------------------------------:|:---------------:|
+| AIX         | 7.1, 7.2                          | PowerPC         |
+| CentOS/RHEL | 6, 7, 8.1, 9                      | x86-64          |
+| Debian      | 9, 10, 11, 12                     | x86-64          |
+| Debian      | 11, 12                            | arm64           |
+| HP-UX       | 11.31+                            | Itanium         |
+| SLES        | 11, 12, 15                        | x86-64          |
+| Solaris     | 11                                | UltraSparc      |
+| Solaris     | 10                                | UltraSparc, x86 |
+| Ubuntu      | 16.04, 18.04, 20.04, 22.04, 24.04 | x86-64          |
+| Ubuntu      | 22.04, 24.04                      | arm64           |
+| Windows     | 2012, 2016, 2019                  | x86-64, x86     |
```

Comment:
Curiously I am wondering about Windows version support. I mean, we build on linux with mingw 32 and 64 bit. Nothing there is specific to any particular version AFAIK so maybe we should update these to reflect newer and/or supported versions or even old versions that are unsupported since that's seems to be our way to a certain extent.

@olehermanse thoughts?